Dockerfile

```diff
@@ -571,6 +571,7 @@ RUN \
   echo "$FFMPEG_SHA256  ffmpeg.tar.bz2" | sha256sum --status -c - && \
   tar xf ffmpeg.tar.bz2 && \
   cd ffmpeg-* && \
+  echo -n "$FFMPEG_VERSION-static *** https://github.com/wader/static-ffmpeg" > VERSION && \
```

Comment:
Know if it is common to change this for ffmpeg builds? thinking if it could be confusing

Reply:
[John Van Sickle](https://johnvansickle.com/ffmpeg/) does and it seems to be the most recognized static builds around